tests/v2/actions/provisioning/creates.go

```diff
@@ -1071,3 +1072,29 @@ func DisableUpdateConfig(client *rancher.Client) {
 		return nil
 	})
 }
+
+// CreateProvisioningRKE1ClusterWithClusterTemplate provisions an rke1 cluster by using the rke1 template and revision ID and other values from the template.
+func CreateProvisioningRKE1ClusterWithClusterTemplate(client *rancher.Client, provider RKE1Provider, clusterTempClusterConfig *clustertemplates.ClusterTemplateConfig, clusterConfig *clusters.ClusterConfig, nodeTemplate *nodetemplates.NodeTemplate) (*management.Cluster, error) {
+	clusterName := namegen.AppendRandomString(provider.Name.String())
```

Comment:
I would avoid sending the entire provider object just to get a string from it. I don't think provider is used elsewhere in this func.